tutorials/schedule-dataflow-jobs-with-cloud-scheduler/schedule-dataflow-jobs-with-cloud-scheduler.md

```diff
@@ -0,0 +1,73 @@
+# Schedule Your Dataflow Batch Jobs With Cloud Scheduler
+
+[Cloud Dataflow](https://cloud.google.com/dataflow) is a managed service for handling 
+both streaming and batch jobs. For your streaming jobs, you just need to launch them once without worrying about operating them afterwards. 
+However, for your batch jobs, you probably need to trigger them based on certain conditions.
+
+In the post, I will show you how you can leverage [cloud scheduler](https://cloud.google.com/scheduler/) to schedule your Dataflow batch jobs. You can find all the code in this [repo](https://github.com/zhongchen/GCP-Demo/tree/master/demos/scheduler-dataflow-demo).
+
+First things first, to be able to run your Dataflow jobs on a regular basis, you need to build your Dataflow templates. 
+Follow the [instructions](https://cloud.google.com/dataflow/docs/guides/templates/creating-templates) to create your templates and save them in a GCS bucket.
+
+![Upload Dataflow templates in a GCS bucket](store_a_template_in_gcs.png)
+
+
+Once you have your templates ready, you can set up cloud schedulers to trigger Dataflow templates.
+
+![Set up your cloud scheduler](set_up_the_cloud_scheduler.png)
+
+
+If you use Terraform, here is one example to define a scheduler.
```

Comment:
Tutorials usually avoid giving multiple paths.  Should the user use the Console (above), or should they use Terraform?   If TF, please give the command they would use to deploy this into their project.

Reply:
sure. I will stick with the TF solution here.

Reply:
I'm OK with the Terraform approach, but there's a lot of setup involved which I think we need to cover.

I've forked your branch, templatized the TF a bit more and added the basic steps here:
https://github.com/jpatokal/community/tree/zhong-cloud-scheduler-dataflow-tutorial

Specific commit: https://github.com/jpatokal/community/commit/488f91e5a8b0caa99775fdd4d41264553f77a519

I don't have the right to commit directly into this PR, but maybe you can pull the commit above from my repo?

Reply:
thanks! I have merged your changes into your branch.